### ISSUE TITLE
shield changes in PT are propagating to regular tabs

### ISSUE THREAD

**Issue by kjozwiak** [brave/browser-laptop#15191]
### Description

Shield settings that are being changed under PT are also affecting the shields of regular tabs which shouldn't be happening. The behaviour should be as follows:

* shield changes in a regular tab will propagate into PT and other tab types
* shield changes in PT should never propagate into regular tabs and other tab types

### Test plan / Steps to Reproduce

  1. install & launch `0.23.205` 81396b3a6454480382ffc476a8b8590fec73717e 
  2. open https://jsfiddle.net/bkf50r8v/13/ in a regular tab
  3. open https://jsfiddle.net/bkf50r8v/13/ in a PT tab
  4. within the PT tab, change FP to `Allow all fingerprinting`
  5. go back into the regular tab and refresh

**Actual result:**
Fingerprinting isn't working for the regular tab but is still enabled via the shields and `Block 3rd Party` is still enabled/selected.

![ptshieldsissue](https://user-images.githubusercontent.com/2602313/45436717-679a4500-b681-11e8-8352-d3bade9d1d08.gif)

**Expected result:**

Changing shields via PT shouldn't be affecting none PT tabs.

**Reproduces how often:**
100% reproducible using the above STR.

### Brave Version

**about:brave info:**
```
Brave: 0.23.205 
V8: 6.9.427.22 
rev: 81396b3a6454480382ffc476a8b8590fec73717e 
Muon: 8.1.4 
OS Release: 17.7.0 
Update Channel: Release 
OS Architecture: x64 
OS Platform: macOS 
Node.js: 7.9.0 
Brave Sync: v1.4.2 
libchromiumcontent: 69.0.3497.92
```

**Reproducible on current live release:**

No, not reproducible using `0.23.107` which is the current live release.

### Additional Information

Reproduced by @srirambv and @LaurenWags on both Win 10 x64 & macOS x64.

**Comment by bsclifton** [comment]
Issue confirmed... must be a Muon issue as there haven't been any changes to browser-laptop (other than the swipe fixes by @darkdh)
https://github.com/brave/browser-laptop/compare/0.23.x...0.23.x-c69?expand=1

**Comment by bsclifton** [comment]
There might be a bug in the C69 branch regarding session
https://github.com/brave/muon/pull/642/files

Basically in browser-laptop, userPref is being updated for the private tab. The code then gets the session object via partition and sets a userPref using `ses.userPrefs.setDictionaryPref`. It appears that this change is bleeding through to the other session/partition (the non-private tab)

**Comment by bsclifton** [comment]
Fixed with 5bfa436fd032e90f168a5353e281cb74fbbcc98f

**Comment by GeetaSarvadnya** [comment]
Verified on Windows 10 x64
•	0.23.206 80a5ac134e251a9cb3565023e2447d6d7b4487a7
•	Muon 8.1.5
•	libchromiumcontent 69.0.3497.92
